deploy/ci/travis/run-e2e-tests.sh

```diff
@@ -21,82 +21,58 @@ export CERTS_PATH=./dev-certs
 # There are two ways of running - building and deploying a full docker-compose deployment
 # or doing a local build and running that with sqlite
 
-E2E_TARGET="e2e-local"
-
 # Single arg to script can change whether we do a quick of full deploy
-RUN_TYPE=$1
-
-if [ "${RUN_TYPE}" == "quick" ]; then
-  echo "Using local deployment for e2e tests"
-  # Quick deploy locally
-  # Start a local UAA - this will take a few seconds to come up in the background
-  docker run -d -p 8080:8080 splatform/stratos-uaa
-
-  # Get go 1.0 and dep
-  curl -sL -o ~/bin/gimme https://raw.githubusercontent.com/travis-ci/gimme/master/gimme
-  chmod +x ~/bin/gimme
-  eval "$(gimme 1.9)"
-  curl https://raw.githubusercontent.com/golang/dep/master/install.sh | sh
-  go version
-  dep version
-  
-  npm run build
-  npm run build-backend
-  # Copy travis config.properties file
-  cp deploy/ci/travis/config.properties src/jetstream/
-  pushd src/jetstream
-  ./jetstream > backend.log &
-  popd
-
-  E2E_TARGET="e2e -- --dev-server-target= --base-url=https://127.0.0.1:5443"
-else
-  echo "Using docker-compose deployment for e2e tests"
-  # Full deploy in docker compose - this is slow
-  # Move the node_modules folder - the docker build will remove it anyway
-  mv ./node_modules /tmp/node_modules
-
-  echo "Building images locally"
-  ./deploy/docker-compose/build.sh -n -l
-  echo "Build Finished"
-  docker images
-
-  echo "Running Stratos in Docker Compose"
-  pushd deploy/ci/travis
-  docker-compose up -d
-  popd
-
-  # The build cleared node_modules, so move back the one we kept
-  #npm install
-  rm -rf ./node_modules
-  mv /tmp/node_modules ./node_modules
-fi
+CAPTURE_VIDEO=$1
```

Comment:
Comment above is now not appropriate to var

Reply:
That was only an issue with the docker-compose build which cleared node_modules

Reply:
Github gave some weird output in the summary view. This was referring to ..
```
# Single arg to script can change whether we do a quick of full deploy
CAPTURE_VIDEO=$1
```

Reply:
Ah yes, I see now